maven-plugin-tools-generators/src/main/java/org/apache/maven/tools/plugin/generator/PluginXdocGenerator.java

```diff
@@ -312,17 +312,15 @@ private void writeGoalAttributes( MojoDescriptor mojoDescriptor, XMLWriter w )
                 w.writeMarkup( format( "pluginxdoc.mojodescriptor.dependencyCollectionRequired", value ) );
                 w.endElement(); //li
             }
-
-            if ( extendedMojoDescriptor.isThreadSafe() )
-            {
-                addedUl = addUl( w, addedUl );
-                w.startElement( "li" );
-                w.writeMarkup( getString( "pluginxdoc.mojodescriptor.threadSafe" ) );
-                w.endElement(); //li
-            }
-
         }
 
+        addedUl = addUl( w, addedUl );
+        w.startElement( "li" );
+        w.writeMarkup( getString( mojoDescriptor.isThreadSafe()
```

Comment:
Why did you replace extended with non-extended?

Reply:
- Please check #50 first.